### ISSUE TITLE
Is there or is it planned a place like "Projects using RactiveJS"?

### ISSUE THREAD

**Issue by ceremcem** [ractivejs/ractivejs.github.io#94]
https://ractive.js.org/integrations/ seems the closest place to look for such a section, but I'm asking just to be sure. I want to introduce you the [ScadaJS](https://github.com/aktos-io/scada.js), a realtime webapp framework that uses RactiveJS in the heart.

**Comment by fskreuz** [comment]
A showcase? Uhm... sort of. Integrations (the tools and plugins) is what we currently have unified into one page. There's a PR for workflows (the various ways Ractive can be used) https://github.com/ractivejs/ractivejs.github.io/pull/73 that's gathering dust that needs polish. We can move to projects right after. It would be a great addition to the site for 1.0.

**Comment by fskreuz** [comment]
This doesn't even need to have it's own page. This can sit directly on the home page. 😄

**Comment by avoto** [comment]
What about listing companies (SaaSj using Ractive? These are not necessary a tool / plugin but still nice to be listed somewhere

**Comment by fskreuz** [comment]
Yep, projects, apps, services etc. Would be nice to have at least 3-5 logos to start with.

**Comment by avoto** [comment]
Herewith a logo of Impact Radius, Inc. We've been using RactiveJS in production since version 0.5. It has been a key technology in delivering mission critical UIs for our large client base worldwide.

**Logo:** 

<img width="200" alt="logo-ir" src="https://user-images.githubusercontent.com/1002464/34328320-714963e2-e891-11e7-8dea-bf6dce7d1dbd.png">

**Website:** http://www.impactradius.com/

**Industry:** Affiliate Marketing

**About:** Our SaaS marketing technology enables marketers to have a singular trusted analytics view into all marketing efforts by collecting granular consumer journey data and marketing costs.
